File: Tree/llRepresentationofBT.java
public class llRepresentationofBT {

    static class Node {
        int data;
        Node left;
        Node right;

        Node(int data) {
            this.data = data;
            this.left = null;
            this.right = null;
        }

    }
    static void inorder(Node root) {
        if (root != null) {
            inorder(root.left);
            System.out.print(root.data + " ");
            inorder(root.right);
        }
    }

    public static void main(String[] args) {
        Node p = new Node(2);
        Node p1 = new Node(1);
        Node p2 = new Node(4);
        p.left = p1;
        p.right = p2;

        inorder(p);
    }
}

File: Tree/llRepresentationofbt.c
#include<stdio.h>
#include<malloc.h>

struct node
{
    int data;
    struct node *left;
    struct node *right;
};

struct node * createNode(int data){
    struct node *p;
    p=(struct node *)malloc (sizeof(struct node));
    p->left=NULL;
    p->right=NULL;
    p->data=data;
    return p;
}


int main(){

    // struct node * p;
    // p=(struct node *)malloc (sizeof(struct node));
    // p->data=2;
    // p->left=NULL;
    // p->right=NULL;

    // struct node * p1;
    // p1=(struct node *)malloc (sizeof(struct node));
    // p1->data=1;
    // p1->left=NULL;
    // p1->right=NULL;

    // struct node * p2;
    // p2=(struct node *)malloc (sizeof(struct node));
    // p2->data=3;
    // p2->left=NULL;
    // p2->right=NULL;

    //Linking left and right node to root node

    struct node * p=createNode(4);
    struct node * p1=createNode(2);
    struct node * p2=createNode(6);

    p->left=p1;
    p->right=p2;
    
    return 0;
}